WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -366,6 +366,11 @@ public async Task<OutputRegistrationResponse> RegisterOutputAsync(OutputRegistra
 
 				var credentialAmount = -request.AmountCredentialRequests.Delta;
 
+				if (StandardScripts.GetTemplateFromScriptPubKey(request.Script) == null)
```

Comment:
```c#
if (!StandardScripts.IsStandardScriptPubKey(request.Script))
``` 
I didn't test it.

Reply:
ah, this is more readable, i found https://github.com/MetacoSA/NBitcoin/blob/236b7decb959f404982ffbbb2b3c9f9401ae3796/NBitcoin/Policy/StandardTransactionPolicy.cs#L249-L257 and missed this nicer method